### ISSUE TITLE
Logz io adding multiple handlers for debug logs

### ISSUE THREAD

**Issue by Aniket-Singla** [logzio/logzio-python-handler#60]
In debug mode the following messages are being printed twice at my console:
Starting to drain 1 logs to Logz.io
Successfully sent bulk of 1 logs to Logz.io!
each time the logs are being sent to logzio.

looks like get_logger function needs to have a check whether the handler(logging.StreamHandler(sys.stdout)) exists previously or not.

**Comment by zoltanctoth** [comment]
Here is a full REPRO so you can see that in debug mode the client will try to send the debug messages to logzio, too, resulting in an infinite loop:

```{python}
import logging
import logzio
import logzio.logger, logzio.handler
import time
import http.client as http_client

for logger_name in ["logzio.logger", "logzio", "requests.packages.urllib3"]:
  if logger_name in logging.Logger.manager.loggerDict:
    print(f"Init: Removing Logger: {logger_name}")
    del(logging.Logger.manager.loggerDict[logger_name])

logzio_logger = logzio.logger.get_logger(debug=True)

formatter = logging.Formatter('LOGZIO_LOG %(asctime)s - %(name)s - %(levelname)s - %(message)s')

# Remove Logzio's default console handler as there will be a new console handler enabled anyhow
# As debug=True will be set on the LogzioHandler
# 
# logzio_logger.handlers[0].setFormatter(formatter)
logzio_logger.handlers = []

logzio_handler = logzio.handler.LogzioHandler(
  token="<<token>>",
  logzio_type='text',
  logs_drain_timeout=3,
  url='https://listener.logz.io:8071',
  debug=True,
  backup_logs=True,
  network_timeout=10.0
)
logzio_handler.setFormatter(formatter)
logzio_logger.addHandler(logzio_handler)

http_client.HTTPConnection.debuglevel = 1
requests_log = logging.getLogger("requests.packages.urllib3")
requests_log.setLevel(logging.DEBUG)
requests_log.propagate = True

logzio_logger.info("INFO TEST")
#logzio_handler.flush()
time.sleep(10)
```

**Comment by tamir-michaeli** [comment]
Fixed in #66

**Comment by tamir-michaeli** [comment]
Inactive - closed.
